components/camel-kafka/src/main/java/org/apache/camel/component/kafka/KafkaConfiguration.java

```diff
@@ -1784,4 +1786,17 @@ public PollOnError getPollOnError() {
     public void setPollOnError(PollOnError pollOnError) {
         this.pollOnError = pollOnError;
     }
+
+    public Long getCommitTimeoutMs() {
+        return commitTimeoutMs;
+    }
+
+    /**
+     * The maximum amount of time, in miliseconds, that the code will wait for a synchronous commit to complete
```

Comment:
amount is a term more used for currency, so I would just say

The maximum time

Also end the description with a dot, all the others do that. Nit picking ;)

Reply:
No worries, I appreciate the review! I'll fix it :)